packages/mask/src/components/DataSource/usePersonaConnectStatus.ts

```diff
@@ -27,13 +27,13 @@ export function usePersonaConnectStatus() {
     return useMemo(() => {
         const id = new ProfileIdentifier(activatedSocialNetworkUI.networkIdentifier, lastRecognized.identifier.userId)
         let connected = false
+        let currentConnectedPersona
         personas.forEach((p) => {
-            p.identifier
-            if (p.linkedProfiles.get(id)) {
-                connected = true
-            }
+            if (!p.linkedProfiles.get(id)) return
+            connected = true
+            currentConnectedPersona = p.publicHexKey
```

Comment:
The `currentConnectedPersona` is ambiguous. 
```suggestion
            currentConnectedPersona = p
```